src/it/resources/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/InputWhitespaceAfterBad.java

```diff
@@ -26,4 +26,8 @@ public void check3 (int...a) { // warn
                 break;
         }
     }
+
+    public void check4() throws java.io.IOException {
```

Comment:
Please update InputWhitespaceAfterGood.java
To show what is good by Google

Reply:
Changes Done